java-logging-appinsights/build.gradle

```diff
@@ -4,8 +4,7 @@ plugins {
 
 def versions = [
   springBoot: '2.1.6.RELEASE',
-  appInsights: '2.3.1',
-  appInsightsStarter: '1.1.2'
+  appInsights: '2.5.0-BETA'
```

Comment:
This version Removed support for <Class> custom instrumentation in AI-Agent.xml which some of projects do use it. Also is beta version recommended for production usage ?

Reply:
🤷‍♂ we need to try it out anyway.
I'll release this as beta